index.js

```diff
@@ -485,6 +485,133 @@ app.post('/api/heartbeat', Validator.body(['coreid', 'event', 'data']).exists(),
   }
 })
 
+app.post('/api/sirenAddressed', Validator.body(['coreid', 'event', 'data']).exists(), async (request, response) => {
```

Comment:
If you don't use `event` or `data`, then you should remove them from the `Validator.body()` call.